test_runner/src/main/kotlin/ftl/Main.kt

```diff
@@ -36,6 +39,10 @@ class Main : Runnable {
     private var printVersion = false
 
     companion object {
+        init {
+//            GoogleApiLogger.logAllToStdout()
```

Comment:
Is this left here for debugging purposes?

Reply:
Yeah, I think it'll be cleaned up as part of https://github.com/TestArmada/flank/issues/446